test/e2e/storage/vsphere/pv_reclaimpolicy.go

```diff
@@ -51,15 +50,19 @@ var _ = utils.SIGDescribe("PersistentVolumes [Feature:ReclaimPolicy]", func() {
 	utils.SIGDescribe("persistentvolumereclaim:vsphere", func() {
 		BeforeEach(func() {
 			framework.SkipUnlessProviderIs("vsphere")
+			Bootstrap(f)
+			nodes := framework.GetReadySchedulableNodesOrDie(c)
+			if len(nodes.Items) < 1 {
+				framework.Skipf("Requires at least %d node", 1)
+			}
+			nodeInfo = TestContext.NodeMapper.GetNodeInfo(nodes.Items[0].Name)
```

Comment:
Looks like this might be a common pattern than can be moved to _ujtils_ or _common_ ?

Reply:
Moved to vsphere_utils.go

added new function

```
func GetReadySchedulableRandomNodeInfo() *NodeInfo {
	nodeList := framework.GetReadySchedulableNodesOrDie(f.ClientSet)
	Expect(nodeList.Items).NotTo(BeEmpty(), "Unable to find ready and schedulable Node")
	rand.Seed(time.Now().Unix())
	nodeInfo := TestContext.NodeMapper.GetNodeInfo(nodeList.Items[rand.Int()%len(nodeList.Items)].Name)
	Expect(nodeInfo).NotTo(BeNil())
	return nodeInfo
}
```